tests/common/util.rs

```diff
@@ -3375,4 +3395,26 @@ mod tests {
         );
         assert!(command.tmpd.is_some());
     }
+
+    #[cfg(all(unix, not(target_os = "macos")))]
+    #[test]
+    fn test_compare_xattrs() {
+        use tempfile::tempdir;
+
+        let temp_dir = tempdir().unwrap();
+        let file_path1 = temp_dir.path().join("test_file1.txt");
+        let file_path2 = temp_dir.path().join("test_file2.txt");
+
+        File::create(&file_path1).unwrap();
+        File::create(&file_path2).unwrap();
+
+        let test_attr = "user.test_attr";
+        let test_value = b"test value";
+        xattr::set(&file_path1, test_attr, test_value).unwrap();
```

Comment:
This line triggers the following error on my machine:
```
thread 'common::util::tests::test_compare_xattrs' panicked at tests/common
called `Result::unwrap()` on an `Err` value: Os { code: 95, kind: Uncategorized, message: "Operation not supported" }
```

Reply:
linux ? mac?

Reply:
On Linux

Reply:
do you have apparmor or selinux?

Reply:
I don't think so 🤔